shared/app/tab-bar/index.render.desktop.js

```diff
@@ -16,6 +16,8 @@ const _icons = {
   [Tabs.profileTab]: {selected: 'icon-nav-people-selected-32', unselected: 'icon-nav-people-32'},
   [Tabs.searchTab]: {selected: 'icon-nav-people-selected-32', unselected: 'icon-nav-people-32'},
   [Tabs.settingsTab]: {selected: 'icon-nav-settings-selected-32', unselected: 'icon-nav-settings-32'},
+  // TODO: Use teams icon when it becomes available.
+  [Tabs.teamsTab]: {selected: 'icon-nav-chat-selected-32', unselected: 'icon-nav-chat-32'},
```

Comment:
these exist today: https://zpl.io/pbL9pM2

Reply:
Left as is, since it turned out they don't exist yet.